docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,299 @@
+For Windows use the following:
+
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+
+</div>
+</div>
+
+## Install Crossplane
+<ul class="nav nav-tabs">
+<li class="active"><a href="#install-tab-helm3" data-toggle="tab">Helm 3 (alpha)</a></li>
+<li><a href="#install-tab-helm2" data-toggle="tab">Helm 2 (alpha)</a></li>
+<li><a href="#install-tab-helm3-master" data-toggle="tab">Helm 3 (master)</a></li>
+<li><a href="#install-tab-helm2-master" data-toggle="tab">Helm 2 (master)</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="install-tab-helm3" markdown="1">
+Use Helm 3 to install the latest official `alpha` release of Crossplane, suitable for community use and testing:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+</div>
+<div class="tab-pane fade" id="install-tab-helm2" markdown="1">
+Use Helm 2 to install the latest official `alpha` release of Crossplane, suitable for community use and testing:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+</div>
+<div class="tab-pane fade" id="install-tab-helm3-master" markdown="1">
+Use Helm 3 to install the latest `master` pre-release version of Crossplane:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-master https://charts.crossplane.io/master/
+helm search repo crossplane-master --devel
+
+# Kubernetes 1.15 and newer versions
+helm install crossplane --namespace crossplane-system crossplane-master/crossplane --version <version> --devel
+```
+
+For example:
+```
+helm install crossplane --namespace crossplane-system crossplane-master/crossplane --version 0.11.0-rc.100.gbc5d311 --devel
+```
+
+</div>
+<div class="tab-pane fade" id="install-tab-helm2-master" markdown="1">
+Use Helm 2 to install the latest `master` pre-release version of Crossplane, which is suitable for testing pre-release versions:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-master https://charts.crossplane.io/master/
+helm search crossplane-master --devel
+
+helm install --name crossplane --namespace crossplane-system crossplane-master/crossplane --version <version>
+```
+
+For example:
+
+```
+helm install --name crossplane --namespace crossplane-system crossplane-master/crossplane --version 0.11.0-rc.100.gbc5d311
+```
+
+</div>
+</div>
+
+## Install Crossplane CLI
+The [Crossplane CLI] adds a set of `kubectl crossplane` commands to simplify common tasks:
```

Comment:
Should this be part of the quickstart? I think a ton of kubectl users don't install plugins, feels like something that could create friction if I think I have to install the plugin before I can try crossplane.

Reply:
The alternative to to create the `ClusterPackageInstall` yaml and apply it, but happy to change it back, so crossplane-cli can be more optional.

Reply:
The alternative to to create the `ClusterPackageInstall` yaml and apply it, but happy to change it back, so crossplane-cli can be more optional.